pkg/kusionctl/cmd/apply/options.go

```diff
@@ -129,8 +131,14 @@ func (o *ApplyOptions) Run() error {
 	return nil
 }
 
-func preview(o *ApplyOptions, planResources *models.Spec,
-	project *projectstack.Project, stack *projectstack.Stack,
+// The preview function will preview for all resources changes
```

Comment:
Write full docstring for public apis. See
https://github.com/KusionStack/KCLVM/blob/eb32a3d87eac48450fa4fb6331470cb735c1cf44/kclvm/tools/src/query/mod.rs#L21
https://github.com/KusionStack/KCLVM/blob/eb32a3d87eac48450fa4fb6331470cb735c1cf44/kclvm/tools/src/query/mod.rs#L21